packages/utils/src/index.ts

```diff
@@ -5,5 +5,4 @@ export * from './log-level';
 export * from './logger';
 export * from './merger';
 export * from './object';
-export * from './s3';
```

Comment:
Exporter `s3` ici transportait du code pour Node jusque dans le front, ce qui pouvait causer le bug `aws-sdk is not defined` lorsqu’on lançait les tests en front